via-router/src/router.rs

```diff
@@ -8,7 +10,10 @@ use crate::path::{self, Param, Pattern, Split};
 ///
 pub struct Route<'a, T>(MatchCond<slice::Iter<'a, MatchCond<T>>>);
 
-pub struct RouteMut<'a, T>(&'a mut Node<T>);
+pub struct RouteMut<'a, T> {
+    node: &'a mut Node<T>,
+    _rc: PhantomData<Rc<()>>,
```

Comment:
This field makes `RouteMut: !Send + !Sync`. Preventing it from being moved into a route handler.